components/script/dom/node.rs

```diff
@@ -2490,6 +2490,11 @@ pub enum ChildrenMutation<'a> {
         next: Option<&'a Node>,
     },
     ReplaceAll { removed: &'a [&'a Node], added: &'a [&'a Node] },
+    /// Mutation for when a Text node's data is modified.
+    /// This doesn't change the structure of the list, which is what the other
+    /// variants' fields are stored for at the moment, so this can just have no
+    /// arguments.
```

Comment:
nit: `this can have no fields` instead? Actually, no big deal, it's understandable.